main.tf

```diff
@@ -127,7 +127,7 @@ data "aws_vpc_endpoint_service" "s3" {
   service = "s3"
 }
 
-resource "aws_vpc_endpoint" "ep" {
+resource "aws_vpc_endpoint" "s3-ep" {
```

Comment:
Let's rename it to `s3` and the one you are adding will be `dynamodb`.

Reply:
Also, we normally do not use dashes inside names.

Reply:
update